### DISCUSSION TITLE
What is the state of linux-drm-syncobj-v1 aka. Explicit Sync in Hyprland?

### DISCUSSION THREAD

**Discussion by CNR0706** [hyprwm/Hyprland#6666] [General]
Now that Hyprland is rapidly moving away from wlroots, I'm a little concerned about the availabillty of Explicit Sync. It seems anything but trivial to implement which has me concerned that Hyprland will be stuck with Implicit Sync for a very long time due to it not being based on wlroots or another library that already supports Explicit Sync.

**Comment by vaxerski** [comment]
it's quite simple to implement, but you gotta wait a bit. It's not that much of a deal as some paint it to be, as nvidia's glitches are mostly gone as the issues seemed to primarily affect xwayland, which got explicit sync

**Reply by CNR0706** [reply]
Alright, thanks for the quick response!
